docs/api/app-locales.md

```diff
@@ -1,4 +1,6 @@
-# Possible Values of `app.getLocale()`
+# Locales
```

Comment:
Yeah, I wrote that in my post, I think it could be but wanted to see if anyone else thought so. So yay! Will update 👍